packages/server/graphql/public/mutations/signUpWithPassword.ts

```diff
@@ -49,13 +48,13 @@ const signUpWithPassword: MutationResolvers['signUpWithPassword'] = async (
   }
   const verificationRequired = await isEmailVerificationRequired(email, dataLoader)
   if (verificationRequired) {
-    const existingVerification = await r
-      .table('EmailVerification')
-      .getAll(email, {index: 'email'})
-      .filter((row: RValue) => row('expiration').gt(new Date()))
-      .nth(0)
-      .default(null)
-      .run()
+    const existingVerification =
+      (await pg
+        .selectFrom('EmailVerification')
+        .selectAll()
+        .where('email', '=', email)
+        .where('expiration', '>', new Date())
+        .executeTakeFirst()) || null
```

Comment:
+1 no need to coerce to null since kysely will return `undefined` which works just as well in this case